src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/data/DomainEvents.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.data
+
+import java.time.OffsetDateTime
+
+data class PrisonerMergeDomainEvent(
```

Comment:
Copied over from "from-nomis-migration" - @petergphillips if we don't need these, we can just remove them